credits.txt

```diff
@@ -20,7 +20,7 @@ Developers
 Major Programming
   AdamKauffman
   Amazinite
-  comnom
+  Frederick Goy IV
```

Comment:
Should this also be moved down to beneath flavious to preserve alphabetical order?